armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -903,24 +903,20 @@ def updateFluenceAndDpa(self, stepTimeInSeconds, blockList=None):
             )
 
         for b in blockList:
-            burnupPeakingFactor = self.getBurnupPeakingFactor(b)
             b.p.residence += stepTimeInSeconds / units.SECONDS_PER_DAY
             b.p.fluence += b.p.flux * stepTimeInSeconds
             b.p.fastFluence += b.p.flux * stepTimeInSeconds * b.p.fastFluxFr
             b.p.fastFluencePeak += b.p.fluxPeak * stepTimeInSeconds * b.p.fastFluxFr
 
             # update detailed DPA based on dpa rate computed at LAST timestep.
-            dpaRateThisStep = b.p.detailedDpaRate
-            newDpaThisStep = dpaRateThisStep * stepTimeInSeconds
-            newDPAPeak = newDpaThisStep * burnupPeakingFactor
-            # track incremental increase for duct distortion interface (and eq)
-            b.p.newDPA = newDpaThisStep
-            b.p.newDPAPeak = newDPAPeak
+            # new incremental DPA increase for duct distortion interface (and eq)
+            b.p.newDPA = b.p.detailedDpaRate * stepTimeInSeconds
+            b.p.newDPAPeak = b.p.detailedDpaPeakRate * stepTimeInSeconds
```

Comment:
Does this somehow account for the fact that the peak location could have changed? For instance if the assembly were shuffled/rotated, it is likely that the peak location is different, and thus just adding to the old value is not necessarily valid.

Reply:
I think fuel handler happens right before depletion on bol. this code is called by depletion. i don't know when another neutron flux solve would happen between the last nodes global flux and shuffling/depletion so i dont know how a different peak would be calculated. if it was, and peakFlux was updated, they could update `detailedDpaPeakRate` accordingly
![image](https://github.com/terrapower/armi/assets/50242044/433dba68-1095-48c4-99e6-c75733d50d22)

Additionally, anytime the global flux interface is called after a shuffle or what nod 773 is called anyway so we should always be using the latest peaking from the latest flux solve. if you don't have a flux solve, how would you update peaking?

ultimately we always recalc flux after shuffling and before any subsequent (non-zero-time) depletion step.

i think at one time we transfer peak fluxes between blocks when we shuffled but that code was deleted, so this would no longer have value

Reply:
I also am not fond of using a burnup peaking factor for dpa... This way if another interface calcs the dpa peaking (setting `detailedDpaPeakRate `) it can get integrated over time here